Examples/NoModules/Chapter 13/Ex13_12/Message.h

```diff
@@ -0,0 +1,24 @@
+#ifndef MESSAGE_H
+#define MESSAGE_H
+
+#include <cstring> // For std::strlen() and std::strcpy()
+
+class Message
+{
+public:
+  explicit Message(const char* text = "")
+    : m_text(new char[std::strlen(text) + 1]) // Caution: include the null character!
+  {
+    std::strcpy(m_text, text);        // Mind the order: strcpy(destination, source)!
```

Comment:
Just saying, but VC++ gives error:
`'strcpy': This function or variable may be unsafe. Consider using strcpy_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.`

Reply:
Of course it does. Yet afaik MSVC is the only compiler that supports `strcpy_s` (and these are not even standard compliant: see https://stackoverflow.com/a/57915261), so I'm not about to use that. In the text I stress plenty of times not to use the C functions, and use std::string instead, don't worry.